### ISSUE TITLE
Error: Built-in hex string literals are obsolete, use `std.conv.hexString!"CC"` instead.

### ISSUE THREAD

**Issue by WebFreak001** [CyberShadow/dcaptcha#2]
The hex strings feature was removed and is giving a compiler error because you are using it in a token string.

```
../../.dub/packages/dcaptcha-1.0.0/dcaptcha/dcaptcha.d(166,8): Error: Built-in hex string literals are obsolete, use `std.conv.hexString!"CC"` instead.
```

This is breaking DFeed

**Comment by CyberShadow** [comment]
Apparently I fixed this three years ago: b9af6e8

**Comment by CyberShadow** [comment]
A GitHub Actions workflow to test building DFeed with and without Dub would be a welcome contribution.
